src/content/platform-integration/ios/app-extensions.md

```diff
@@ -225,6 +225,8 @@ use an iOS simulator to test your extension in debug mode.
    [Adding a Flutter Screen][]. For example, you can display a
    specific route in your Flutter app within a share extension.
 
+1. Cancel the request in `viewDidDisappear`, which will be called when the share extension is dismissed by gesture. 
```

Comment:
I don't see a "share" extension. Does you mean share_plus?
If so, we should say `share_plus`.

```suggestion
1. Cancel the request in `viewDidDisappear`,
    which is called when the `share` extension is dismissed with a gesture. 
```

Reply:
I mean "share extension" which is one kind of app extensions. We use "share extension" as the example in this section: https://flutter-docs-prod--pr11882-share-extension-memory-leak-73qwnwz8.web.app/platform-integration/ios/app-extensions#creating-app-extension-uis-with-flutter

Reply:
OK! I didn't understand what a "share extension" was. Now that I do, "share" shouldn't be codefonted, _but_ when you first mention "share extension" I would like to see a quick note or parenthetical that says what that is. Something simple and quick. I don't mind who lands/modifies this PR, whether it's @hellohuanlin or @antfitch, but I'd like this clarification.

Reply:
thanks for the comment. I think we should be fine here, as "share extension" is an official term by apple, so whoever wants to implement this feature should probably already know about it (also we have pictures right in this section so should be self-explanatory). It may be a good idea to link to apple's official doc tho.